src/components/CompetenceGoalTab.tsx

```diff
@@ -173,7 +174,7 @@ export const CompetenceItem = ({ item, isOembed, showLinks = false }: Competence
             </InnerList>
           ) : (
             <CoreElementWrapper>
-              <Text>{element.text}</Text>
+              <Text>{parse(element.text)}</Text>
```

Comment:
Hvis det er en fare for at dette kan inneholde `p` burde `Text` asChildes til en `div`. Hvis ikke vil vi ende opp med p i p